project-koku/koku
# Adding OCI changes to onboard to new ochestrator

Fixes [COST-2683](https://issues.redhat.com/browse/COST-2683)

Changes proposed in this PR:
* Onboard OCI to new ochestrator flow


Testing Instructions:
1. Add OCI source
2. Ingest data for multiple months
3. Expect to see data populated correctly in all month bound tables

---
Additional Context:
Reach out to Luke for adding OCI as a source (Need to use a real source since this is really for the non local downloader)
